File: Hotel.API/Reservations/Responses/ReservationResponse.cs
namespace Hotel.API.Reservations.Responses;

public record ReservationResponse(
    Guid Id,
    Guid CreatorId,
    Guid RoomId,
    Guid RatePlanId,
    DateTime StartDate,
    DateTime EndDate,
    DateTime? ArrivalTime,
    DateTime CreatedAt,
    List<Guid> GuestIds);

File: Frontend/Admin/admin-panel/src/app/feature/reservations/models/responses/reservation.response.ts
export interface ReservationResponse {
  id: string;
  cycleIdentifier: string;
  creatorId: string;
  roomId: string;
  ratePlanId: string;
  startDate: string;
  endDate: string;
  arrivalTime: string | null;
  createdAt: string;
  guestIds: string[];
}